packages/JetSelector/JetSelector.C

```diff
@@ -94,7 +94,7 @@ void JetSelector::Initialise(){
   }
   else if(gSelection == iWZSelec){
     taggerName="CSVv2";
-    stringWP = "Loose";
+    stringWP = "Medium";
```

Comment:
Why? Any benefit?

Reply:
The b-jet veto in:
[https://github.com/vischia/cmgtools-lite/blob/wz80x/TTHAnalysis/python/plotter/wzsm/cuts_wzsm.txt](https://github.com/vischia/cmgtools-lite/blob/wz80x/TTHAnalysis/python/plotter/wzsm/cuts_wzsm.txt) 
It seems to be applied for the medium WP. Nonetheless I did not delve in the FriendTree creation code so I might be mistaken.

Reply:
Ah, I had inverted red and white, I thought you were changing to Loose.

I think I had tried Loose at some point, but I was not satisfied by the outcome. I will re-run the test in the main framework